SwiftCompilerSources/Sources/SIL/Instruction.swift

```diff
@@ -98,6 +98,10 @@ public class Instruction : ListNode, CustomStringConvertible, Hashable {
     }
   }
 
+  final public var mayReleaseOrReadRefCount: Bool {
```

Comment:
I would prefer to have two separate properties: `mayRelease` and `mayReadRefCount`.
`mayReadRefCount` can be implemented in swift (no need to bridge it).